roles/wordpress-setup/tasks/database.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Create database of sites
+  mysql_db: name="{{ item.value.env.db_name | default(item.key) }}"
+            state=present
+            login_host="{{ item.value.env.db_host | default('localhost') }}"
+            login_user="{{ mysql_user }}"
+            login_password="{{ mysql_root_password }}"
+  with_dict: wordpress_sites
+  when: item.db_create | default(True)
+
+- name: Create/assign database user to db and grant permissions
+  mysql_user: name="{{ item.value.env.db_user }}"
+              password="{{ item.value.env.db_password }}"
+              priv="{{ item.value.env.db_name | default(item.key) }}.*:ALL"
+              state=present
+              login_host="{{ item.value.env.db_host | default('localhost') }}"
+              login_user="{{ mysql_user }}"
+              login_password="{{ mysql_root_password }}"
+  with_dict: wordpress_sites
+  when: item.db_create | default(True)
+
+- name: Copy database dump
+  copy: src="{{ item.value.db_import }}" dest=/tmp
+  with_dict: wordpress_sites
+  when: item.db_import | default(False)
```

Comment:
This has to be `item.value.db_import` or it'll never fire. It appears a couple places in this file.